package/MDAnalysis/core/Selection.py

```diff
@@ -943,23 +881,51 @@ class SelectionParser:
     FULLSELGROUP = 'fullgroup'
 
     classdict = dict([
-        (ALL, AllSelection), (NOT, NotSelection), (AND, AndSelection), (OR, OrSelection),
-        (SEGID, SegmentNameSelection), (RESID, ResidueIDSelection), (RESNUM, ResnumSelection),
-        (RESNAME, ResidueNameSelection), (NAME, AtomNameSelection), (ALTLOC, AltlocSelection),
-        (TYPE, AtomTypeSelection), (BYRES, ByResSelection),
-        (BYNUM, ByNumSelection), (PROP, PropertySelection),
-        (AROUND, AroundSelection), (SPHLAYER, SphericalLayerSelection), (SPHZONE, SphericalZoneSelection),
-        (CYLAYER, CylindricalLayerSelection), (CYZONE, CylindricalZoneSelection),
-        (POINT, PointSelection), (NUCLEIC, NucleicSelection), (PROTEIN, ProteinSelection),
-        (BB, BackboneSelection), (NBB, NucleicBackboneSelection),
-        (BASE, BaseSelection), (SUGAR, NucleicSugarSelection),
-        #(BONDED, BondedSelection), not supported yet, need a better way to walk the bond lists
```

Comment:
Because BONDED was commented out here, there's no link between the string "BONDED" and the class above